iphone/Classes/TiUISearchBar.m

```diff
@@ -141,11 +141,22 @@ -(void)setBarColor_:(id)value
 -(void)setBackgroundImage_:(id)arg
 {
     UIImage *image = [self loadImage:arg];
-    [[self searchBar] setBackgroundImage:image];
+    if([TiUtils isIOS7OrGreater]) {
```

Comment:
iOS7 is our minimum supported version, so this if condition will always return true. You can remove it